headers/types/common/common.h

```diff
@@ -144,20 +144,13 @@ struct dialog_box {
 };
 ASSERT_SIZE(struct dialog_box, 224);
 
-// Structure for dialog boxes with portraits?
+// Represent a portrait that appear inside a dialogue box
 struct portrait_box {
```

Comment:
> Also, I think portrait_box is the "kaomodo" struct in this PR: https://github.com/UsernameFodder/pmdsky-debug/pull/21/files

Hmm I think you're very likely correct. Could we maybe copy the fields over from https://github.com/UsernameFodder/pmdsky-debug/pull/21 (with comments saying that they need verification)?

Reply:
Actually, on second thought, copying over just that stuff from #21 probably doesn't make sense out of context. We can always port that stuff another day, or try to get the original PR merged.

Reply:
I agree with that (thought kaomodo may not be an appropriate name for this struct, given that’s also how the file format is called)